pkg/kp/rcstore/consul_store.go

```diff
@@ -168,6 +169,10 @@ func (s *consulStore) WatchNew(quit <-chan struct{}) (<-chan []fields.RC, <-chan
 func (s *consulStore) kvpToRC(kvp *api.KVPair) (fields.RC, error) {
 	rc := fields.RC{}
 	err := json.Unmarshal(kvp.Value, &rc)
+
+	if rc.Manifest == nil {
```

Comment:
i'm not sure if this is the right fix actually, it can cause a `List()` operation to fail if one rc has a nil manifest. Probably better to address line 343, but i'll have to look at the callers of that